website/content/api-docs/auth/ldap.mdx

```diff
@@ -74,7 +74,8 @@ This endpoint configures the LDAP auth method.
 - `userfilter` `(string: "")` – An optional LDAP user search filter.
   The template can access the following context variables: UserAttr, Username.
   The default is `({{.UserAttr}}={{.Username}})`, or `({{.UserAttr}}={{.Username@.upndomain}})`
-  if `upndomain` is set.
+  if `upndomain` is set. In order for the `userfilter` to be applied,
```

Comment:
nit: I wonder if it would be worthwhile having a section specifically showing the different configurations that must be set for userfilter to apply? We could add a section here https://developer.hashicorp.com/vault/docs/auth/ldap and link to it from this API doc?